src/FastBoard.cpp

```diff
@@ -392,7 +421,10 @@ std::vector<bool> FastBoard::calc_reach_color(int col) {
     return bd;
 }
 
-// Needed for scoring passed out games not in MC playouts
+/*
+   The score is from the point of view of the black player. A negative score means white is leading.
+   @return score needed for scoring passed out games not in Monte-Carlo play-outs
```

Comment:
It returns the area score. The explanation why this is needed is not related to the return value, so put it in the function description.

Reply:
I removed the @return tag and updated the function description.